logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -352,6 +355,12 @@ internal class ConversationDataSource internal constructor(
                 .map { it.map(conversationMapper::fromDaoModel) }
         }
 
+    override suspend fun getProteusTeamConversationsReadyForFinalisation(teamId: TeamId): Either<StorageFailure, Flow<List<QualifiedID>>> =
```

Comment:
IMO `Flow` here is redundant. We don't observe the changes, always call `.first()`. So maybe just change it to 
```
suspend fun getProteusTeamConversationsReadyForFinalisation(teamId: TeamId): Either<StorageFailure, List<QualifiedID>>
```
Also we'll need to update `ConversationDAO.getAllProteusTeamConversationsReadyToBeFinalised`

Reply:
True, let's get rid of the flow.

Reply:
Gone